[
  "test_authuser_kerb.py::test_authssotoken",
  "test_authuser_kerb.py::test_device_login",
  "test_authuser_kerb.py::test_kerblogin_private",
  "test_authuser_kerb.py::test_kerblogin_private_wrong_secret",
  "test_authuser_kerb.py::test_kerblogin_public",
  "test_authuser_kerb.py::test_kerblogin_public_not_auth",
  "test_fastapi.py::test_secure_auth",
  "test_fastapi.py::test_secure_auth_no_client_id[-False-200]",
  "test_fastapi.py::test_secure_auth_no_client_id[-True-500]",
  "test_fastapi.py::test_secure_auth_no_client_id[None-False-200]",
  "test_fastapi.py::test_secure_auth_no_client_id[None-True-500]",
  "test_fastapi.py::test_secure_noauth",
  "test_fastapi.py::test_secure_wrong_aud",
  "test_fastapi.py::test_session_auth",
  "test_fastapi.py::test_session_auth_customstore",
  "test_fastapi.py::test_unsecure",
  "test_flask.py::test_secure_auth",
  "test_flask.py::test_secure_noauth",
  "test_flask.py::test_secure_session_auth",
  "test_flask.py::test_unsecure"
]